tests/tests.py

```diff
@@ -176,17 +178,45 @@ def test_empty_options(self):
                 options=[],
                 mode='APPEND_OPTION')
 
+class TestClassify(unittest.TestCase):
+    def test_successful_classify(self):
+        prediction = co.classify('medium', ["this restaurant is great!"], [ClassifyExample("this restaurant is bad", "negative"), ClassifyExample("this place is amazing!", "positive")])
```

Comment:
i am not a big fan of ClassifyExample("this restaurant is bad", "negative"). Python is generally less/not typed.

if it could be [("this restaurant is bad", "negative"), ("This place is amazing")] or

just a dict
{
   "this restaurant is bad":  "negative",
   "this place is amazing!":  "positive"
}

Reply:
it makes more sense that it is a `Dict[str, str]` as the ordering if the examples does not really matter

Reply:
Hmm yea I'll run it by product + ivan on Monday cc: @1vn